From 88a97229350010d73cee0ff38da0dbbe0a449dd2 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 12 Sep 2022 11:20:59 -0700
Subject: [PATCH] fwupd: Drop unsupported xml_declaration parameter

This is not needed on Python3.7

Fixes
wupd-1.8.4/contrib/generate-gresource-xml.py", line 26, in <module>
    f.write(ET.tostring(root, "utf-8", xml_declaration=True))
TypeError: tostring() got an unexpected keyword argument 'xml_declaration'

Upstream-Status: Submitted [https://github.com/fwupd/fwupd/pull/5025]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 contrib/generate-gresource-xml.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/generate-gresource-xml.py b/contrib/generate-gresource-xml.py
index 235a7700..4429f988 100755
--- a/contrib/generate-gresource-xml.py
+++ b/contrib/generate-gresource-xml.py
@@ -23,6 +23,6 @@ for fn in sorted(sys.argv[2:]):
         n_file.set("preprocess", "xml-stripblanks")
     n_file.set("alias", os.path.basename(fn))
 with open(sys.argv[1], "wb") as f:
-    f.write(ET.tostring(root, "utf-8", xml_declaration=True))
+    f.write(ET.tostring(root, "utf-8"))
 
 sys.exit(0)
-- 
2.37.3

